src/imgui/renderer/imgui_core.cpp

```diff
@@ -51,8 +51,14 @@ void Initialize(const ::Vulkan::Instance& instance, const Frontend::WindowSDL& w
     io.ConfigFlags |= ImGuiConfigFlags_NavEnableGamepad;
     io.ConfigFlags |= ImGuiConfigFlags_DockingEnable;
     io.DisplaySize = ImVec2((float)window.getWidth(), (float)window.getHeight());
-    io.IniFilename = SDL_strdup(config_path.string().c_str());
-    io.LogFilename = SDL_strdup(log_path.string().c_str());
+
+    using native_char = std::filesystem::path::value_type;
+    io.IniFilename = new char[config_path.native().size() * sizeof(native_char)];
+    io.LogFilename = new char[log_path.native().size() * sizeof(native_char)];
+    std::memcpy((void*)io.IniFilename, config_path.native().c_str(),
+                config_path.native().size() * sizeof(native_char));
+    std::memcpy((void*)io.LogFilename, log_path.native().c_str(),
+                log_path.native().size() * sizeof(native_char));
```

Comment:
Imgui expects to be UTF8 and internally converts to Wide Char.
```suggestion
    auto path = config_path.u8string();
    char* config_file_buf = new char[path.size() + 1];
    config_file_buf[path.copy((char8_t*)config_file_buf, path.size())] = '\0';
    io.IniFilename = config_file_buf;

    path = log_path.u8string();
    char* log_file_bug = new char[path.size() + 1];
    log_file_bug[path.copy((char8_t*)log_file_bug, path.size())] = '\0';
    io.LogFilename = log_file_bug;
```

Also change the Shutdown below here to use delete instead of SDL_free
```cpp
    const auto ini_filename = io.IniFilename;
    const auto log_filename = io.LogFilename;

    Vulkan::Shutdown();
    Sdl::Shutdown();
    DestroyContext();

    delete[] ini_filename;
    delete[] log_filename;
```